.github/workflows/main.yml

```diff
@@ -20,7 +20,7 @@ on:
       - submitted
 
 env:
-  XCLIM_TESTDATA_BRANCH: v2023.5.4
+  XCLIM_TESTDATA_BRANCH: v2023.6.8
 
```

Comment:
Maybe this part too concerns the PR changing wsgsmax -> sfcWindmax ?

Reply:
Yep, we have checks to ensure that both the `main` and latest tag of testing data are in sync. This was related to #1358.